assets/js/base/components/product-list/style.scss

```diff
@@ -55,217 +55,14 @@
-			content: ".";
-			display: inline-block;
-			width: 3em;
-		}
-	}
-}
-
-.wc-block-grid__product-add-to-cart {
-	word-break: break-word;
-	white-space: normal;
-	a,
-	button {
-		word-break: break-word;
-		white-space: normal;
-		margin: 0 auto !important;
-		display: inline-flex;
-		justify-content: center;
-
-		&.loading {
-			opacity: 0.25;
-		}
-
-		&::after {
-			margin-left: 0.5em;
-			display: inline-block;
-		}
-
-		&.added::after {
-			font-family: WooCommerce; /* stylelint-disable-line */
-			content: "\e017";
-		}
-
-		&.loading::after {
-			font-family: WooCommerce; /* stylelint-disable-line */
-			content: "\e031";
-			animation: spin 2s linear infinite;
-		}
-
-		.is-loading & {
-			@include placeholder();
-			min-width: 7em;
-		}
-	}
-}
-
-.wc-block-grid__product-rating {
-	display: block;
-
-	.wc-block-grid__product-rating__stars,
-	.star-rating {
-		overflow: hidden;
-		position: relative;
-		width: 5.3em;
-		height: 1.618em;
-		line-height: 1.618;
-		font-size: 1em;
-		/* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
-		font-family: star;
-		font-weight: 400;
-		margin: 0 auto;
-		text-align: left;
-
-		&::before {
-			content: "\53\53\53\53\53";
-			top: 0;
-			left: 0;
-			right: 0;
-			position: absolute;
-			opacity: 0.5;
-			color: #aaa;
-			white-space: nowrap;
-		}
-		span {
-			overflow: hidden;
-			top: 0;
-			left: 0;
-			right: 0;
-			position: absolute;
-			padding-top: 1.5em;
-		}
-		span::before {
-			content: "\53\53\53\53\53";
-			top: 0;
-			left: 0;
-			right: 0;
-			position: absolute;
-			color: #000;
-			white-space: nowrap;
-		}
-	}
-}
-
-.wc-block-grid__product-onsale {
-	border: 1px solid #43454b;
-	color: #43454b;
-	background: #fff;
-	padding: 0.202em 0.6180469716em;
-	font-size: 0.875rem;
-	text-align: center;
-	text-transform: uppercase;
-	font-weight: 600;
-	display: inline-block;
-	width: auto;
-	border-radius: 3px;
-	z-index: 9;
-	position: relative;
-	margin: $gap-smaller auto;
-}
-
-.editor-styles-wrapper .wc-block-grid__products .wc-block-grid__product .wc-block-grid__product-image,
-.wc-block-grid__product-image {
-	.wc-block-grid__product-onsale {
-		&.wc-block-grid__product-onsale--alignleft {
-			position: absolute;
-			left: $gap-smaller/2;
-			top: $gap-smaller/2;
-			right: auto;
-			margin: 0;
-		}
-		&.wc-block-grid__product-onsale--aligncenter {
-			position: absolute;
-			top: $gap-smaller/2;
-			left: 50%;
-			transform: translateX(-50%);
-			margin: 0;
-		}
-		&.wc-block-grid__product-onsale--alignright {
-			position: absolute;
-			right: $gap-smaller/2;
-			top: $gap-smaller/2;
-			left: auto;
-			margin: 0;
-		}
-	}
-}
-
-// Element spacing.
-.wc-block-grid__product {
-	.wc-block-grid__product-image,
-	.wc-block-grid__product-title,
-	.wc-block-grid__product-price,
-	.wc-block-grid__product-rating,
-	.wc-block-grid__product-add-to-cart {
-		margin-top: 0;
-		margin-bottom: $gap-small;
-	}
-}
-
+.editor-styles-wrapper .wc-block-grid,
```

Comment:
If this fix is only for 2020 (44bf3aba2d070dbd47bbbf0b2b19d2f20828ba7d), could we instead do `.theme-twentytwenty .wc-block-grid { ... }`? This way we avoid different selector specificity in the editor _vs._ frontend.